src/components/SectorChangeCountDown.tsx

```diff
@@ -22,8 +22,7 @@ export default observer(function SectorChangeCountDown(/* properties */) {
   const { timeToNextConfiguration, shouldShowNextConfiguration } = configurationStore;
   // eslint-disable-next-line @typescript-eslint/unbound-method
   const { toggleShowNextSectorsConfiguration, showNextSectorsConfiguration } = cwpStore;
-
-  if (timeToNextConfiguration > 6_000_000) {
```

Comment:
Oops, forgot to change that (it was for debug).

Reply:
Aha, I thought maybe you just pressed too many zeros